pkg/network/tracer/tracer_test.go

```diff
@@ -89,7 +89,10 @@ func setupTracer(t testing.TB, cfg *config.Config) *Tracer {
 
 func (s *TracerSuite) TestGetStats() {
 	t := s.T()
-	httpSupported := httpSupported()
+	if !httpSupported() {
```

Comment:
Aren't there still stats to be tested if http is not enabled?

Reply:
true, reverted